src/components/app/MenuButtons.js

```diff
@@ -589,6 +604,69 @@ const MenuButtons = ({
   predictUrl,
 }: MenuButtonsProps) => (
   <div className="menuButtons">
+    <ButtonWithPanel
+      className="menuButtonsOpenMetainfo"
+      label="Profile info"
+      panel={
+        <ArrowPanel className="arrowPanelOpenMetainfo">
+          <h2 className="arrowPanelSubTitle">Timing</h2>
+          <div className="arrowPanelSection">
+            <span>
+              <span className="metainfolabel">startTime: </span>
+              {_formatDate(profile.meta.startTime)}
+            </span>
+            <br />
+            <span>
+              <span className="metainfolabel">Interval:</span>{' '}
+              {profile.meta.interval}s
+            </span>
+            <br />
+            <span>
+              <span className="metainfolabel">Profile Version:</span>{' '}
+              {profile.meta.preprocessedProfileVersion}
+            </span>
+          </div>
+          <h2 className="arrowPanelSubTitle">Application</h2>
+          <div className="arrowPanelSection">
+            <span>
+              <span className="metainfolabel">Name:</span>{' '}
+              {profile.meta.product}
+            </span>
+            <br />
+            <span>
+              <span className="metainfolabel">Version:</span>{' '}
+              {profile.meta.misc}
+            </span>
+            <br />
+            <span>
+              <span className="metainfolabel">Build ID:</span>{' '}
+              {profile.meta.appBuildID}
+            </span>
+            <br />
```

Comment:
The `<br />` is only necessary if the extensions list is non-empty. Otherwise it will cause a blank line.

I think if you use grid layout, you don't need the `<br>`s, and you won't have spurious blank lines.